ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -1564,13 +1564,13 @@ def getFundamentalFeatures(self, requestedFeatures, featureTemplate=None):
     for target, arma in self.armaResult.items():
       # sigma
       feature = featureTemplate.format(target=target, metric='arma', id='std')
-      features[feature] = np.sqrt(arma.sigma2)
+      features[feature] = np.sqrt(arma.params[arma.param_names.index('sigma2')])
       # autoregression
-      for p, val in enumerate(arma.arparams):
+      for p, val in enumerate(-arma.polynomial_ar[1:]):
```

Comment:
I think this was discussed in a comment on a different PR, but would you mind adding a small comment in the code explaining why we are taking the negative AR 1-nth index coef?